backend/src/controllers/admins.js

```diff
@@ -40,6 +40,21 @@ export async function getAdmins(req, res) {
     }
 }
 
+export async function deleteAdmin(req, res) {
+    try{
+        const {user} = req.params
+        const adm_deletado = await Admin.findOneAndDelete({user: user});
+        if(!adm_deletado) {
+            return res.status(401).send({message: 'Admin não existe.'});
```

Comment:
Shouldn't the code here be 404?

Reply:
I think the editPassword controller also uses 401, maybe change that one too?

Reply:
Perhaps the scope of this PR can also include adding the "admin username already in use" check in the creation endpoint.

Reply:
done